src/content/en/fundamentals/accessibility/index.md

```diff
@@ -1,8 +1,355 @@
+users.
+>
+>This is a broad range, from someone like my wife, who doesn't have any corneas
+&mdash; so while she can basically see things she has a hard time reading print
+and is considered legally blind &mdash; to someone who might have just poor
+vision and needs to wear very strong prescription glasses.
+>
+>There's a huge range, and so naturally there's a big range of accommodations
+that people in this category use: some do use a screen reader or a braille
+display (I've even heard of one woman who reads braille displayed on-screen
+because it's easier to see than printed text), or they might use text-to-speech
+technology without the full screen reader functionality, or they might use a
+screen magnifier which zooms in on part of the screen, or they might just use
+their browser zoom to make all the fonts bigger. They might also use
+high-contrast options like an operating system high-contrast mode, a
+high-contrast browser extension or a high-contrast theme for a website.
+
+<figure>
+  <img src="imgs/high-contrast.png" alt="High-contrast mode">	
+  <figcaption>High-contrast mode</figcaption>
+</figure>
+
+>A lot of users even use a combination of these, like my friend Laura who uses a
+combination of high-contrast mode, browser zoom and text-to-speech.
+>
+>Low vision is something a lot of people can relate to. For a start, we all
+experience deteriorating vision as we age, so even if you haven't experienced it
+there's a good chance you've heard your parents complain about it. But a lot of
+people experience the frustration of taking their laptop out by a sunny window
+only to find they suddenly can't read anything! Or anyone who's had laser
+surgery or maybe just has to read something from across the room might have used
+one of those accommodations I mentioned. So I think it's pretty easy for
+developers to have some empathy for low-vision users.
+>
+>Oh, and I shouldn't forget to mention people with poor color vision &mdash;
+about 9% of males have some form of color vision deficiency! Plus about 1% of
+females. They may have trouble distinguishing red and green, or yellow and blue.
+Think about that the next time you design form validation.
+
+ *What about motor impairments?*
+
+>Yes, motor impairments, or dexterity impairments. This group ranges all the way
+from those who would prefer not to use a mouse, because perhaps they've had some
+RSI or something and find it painful, to someone who may be physically paralyzed
+and have limited range of motion for certain parts of their body. Motor impaired
+users may use a keyboard, switch device, voice control, or even an eye-tracking
+device to interact with their computer.
+
+<figure>
+  <img src="imgs/switch-device.png" alt="A person using a switch device with their foot">	
+  <figcaption>A switch device</figcaption>
+</figure>
+
+<figure>
+  <img src="imgs/eye-tracking.png" alt="A person using an eye tracking device">	
+  <figcaption>An eye tracking device</figcaption>
+</figure>
+
```

Comment:
add a `<div class="clearfix"></div>` to clear the float and ensure the next bit fits properly